Flow.Launcher/ViewModel/MainViewModel.cs

```diff
@@ -1401,6 +1401,16 @@ public async void Hide()
                         await Task.Delay(100);
                     LastQuerySelected = false;
                     break;
+                case LastQueryMode.ActionKeywordPreserved or LastQueryMode.ActionKeywordSelected:
+                    var newQuery = _lastQuery.ActionKeyword;
+                    if (!string.IsNullOrEmpty(newQuery))
+                        newQuery += " ";
+                    ChangeQueryText(newQuery);
+                    if (Settings.UseAnimation)
+                        await Task.Delay(100);
```

Comment:
Why do we wait here?

Reply:
Ok because using animation, think previously other options had some issues when animation is in process.

Reply:
I'm actually not sure. It's mentioned here https://github.com/Flow-Launcher/Flow.Launcher/blob/0506f095b34bc10d34eaa54dff171ff5f803ef08/Flow.Launcher/ViewModel/MainViewModel.cs#L1390-L1403 and I added it in my code because this was present in all three other `case` branches, so it probably solves some old issue I don't know about.